plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/transactionlog/TestCheckpointMetadataEntry.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
We want to verify regressions with query based tests instead of unit tests. 
Please update existing integration tests or product tests.